docs/installation-guide-azure.md

```diff
@@ -50,7 +50,7 @@ Create service principal with permissions limited to resource group.
 
 ```
 export SUBSCRIPTION_ID=$(az account list | jq '.[0].id' | sed 's/\"//g')
-az ad sp create-for-rbac --name=${NAME}-sp --role="Contributor" --scopes="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${NAME}"
+az ad sp create-for-rbac --name=${NAME}-sp --role="Contributor" --scopes="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${NAME}" --years 10
```

Comment:
by default it's only one year, which is not enough and we don't have procedure to roll accounts at the moment.